addons-cluster/redis/templates/_helpers.tpl

```diff
@@ -188,6 +197,56 @@ app.kubernetes.io/managed-by: "kubeblocks"
 apps.kubeblocks.io/component-name: "redis"
 {{- end }}
 
+{{/*
+Redis Cluster schedulingPolicy
+*/}}
+{{- define "redis-cluster.schedulingPolicy" }}
+schedulingPolicy:
+  affinity:
+    podAntiAffinity:
```

Comment:
Why set both `preferredDuringSchedulingIgnoredDuringExecution` and `requiredDuringSchedulingIgnoredDuringExecution` at the same time? Moreover, their `labelSelector` and `topologyKey` look identical, so the result of this setup is that `requiredDuringSchedulingIgnoredDuringExecution` takes precedence, making `preferredDuringSchedulingIgnoredDuringExecution` redundant.

Reply:
Thanks @zjx20 
Great. The `preferredDuringSchedulingIgnoredDuringExecution` setting has no practical effect in this case. For Redis deployments, it's more appropriate to use only `preferredDuringSchedulingIgnoredDuringExecution`.